lib/matplotlib/ticker.py

```diff
@@ -2154,6 +2156,19 @@ def is_decade(x, base=10, *, rtol=1e-10):
     return is_close_to_int(lx, atol=rtol)
 
 
+def _is_decade(x, *, base=10, rtol=None):
+    """Return True if *x* is an integer power of *base*."""
+    if not np.isfinite(x):
+        return False
+    if x == 0.0:
+        return True
+    lx = np.log(abs(x)) / np.log(base)
+    if rtol:
```

Comment:
```suggestion
    if rtol is not None:
```

Relying on the "falsy-ness" of `rtol` means that you can pass an empty list and it would "work"!